vars/createOpenshiftDockerSecret.groovy

```diff
@@ -0,0 +1,19 @@
+def call(String dockerServer, String username, String password, String email, String secretName='dockerhub') {
+    openshift.withCluster() {
+
+        if (openshift.selector('secrets').names().contains("secret/${secretName}" as String)) {
```

Comment:
In the check here, I had to explicitly say `as String` for the `"secret/${secretName}"`, because otherwise it's a `GString` and the `List.contains(...)` doesn't find it.